huaweicloud/services/vpc/resource_huaweicloud_vpc_route.go

```diff
@@ -284,18 +285,17 @@ func getDefaultRouteTable(client *golangsdk.ServiceClient, vpcID string) (string
 	return defaultID, nil
 }
 
-func parseResourceID(id string) (string, string) {
+func parseResourceID(id string) (tableID, destination string) {
 	parts := strings.SplitN(id, "/", 2)
 	if len(parts) != 2 {
-		return "", ""
+		return
 	}
 
-	return parts[0], parts[1]
+	tableID, destination = parts[0], parts[1]
+	return
 }
 
-func resourceVpcRTBRouteImportState(_ context.Context, d *schema.ResourceData,
-	meta interface{}) ([]*schema.ResourceData, error) {
-
+func resourceVpcRTBRouteImportState(_ context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
```

Comment:
nothing is set in this function. 
it's better to merge the `parseResourceID` and `resourceVpcRTBRouteImportState`

Reply:
`parseResourceID` is also called by other functions.